src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -66,7 +66,7 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('collectors.xml');
         $loader->load('guard.xml');
 
-        if ($container->hasParameter('kernel.debug') && $container->getParameter('kernel.debug')) {
+        if ($container->getParameter('kernel.debug')) {
             $loader->load('security_debug.xml');
```

Comment:
@javiereguiluz I tried this PR and the issue still exists :(

https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/SecurityBundle/Resources/config/collectors.xml#L13: The `data_collector.security` service ("which is always loaded") require `debug.security.access.decision_manager`, so when `$container->getParameter('kernel.debug')` is `false` then:
```
  [Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException]
  The service "profiler" has a dependency on a non-existent service "debug.security.access.decision_manager".
```

The road should be here https://github.com/symfony/symfony-standard/issues/968#issuecomment-223022665:
> ... It should be based on whether the profiler is enabled instead of being based on the debug mode ...